runtime/lua/vim/lsp/rpc.lua

```diff
@@ -608,6 +608,73 @@ function M.connect(host, port)
   end
 end
 
+---@class Dispatchers
+---@field notification function
+---@field server_request function
+---@field on_exit function
```

Comment:
In line with my comment about `function` versus `fun`, consider using the latter to include parameter and return types.